scss/dpt_classic.scss

```
@@ -587,7 +587,26 @@ div#dialogFrame {
   overflow-y: auto;
   padding-top: 40px;
 }
-
+#form.dialog-form{
+    position:absolute;
+    width:40%;
+    height:12em;
+    left:0;
+    right:0;
+    top:0;
+    bottom:0;
+    padding:1em;
+    margin:auto;
+    
+}
+
+#form.dialog-form form{
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    justify-content:space-between;
+    height:100%;
+}
```

Let's use SASS nesting here.

Example:
```suggestion
#form.dialog-form{
    position:absolute;
    width:40%;
    height:12em;
    left:0;
    right:0;
    top:0;
    bottom:0;
    padding:1em;
    margin:auto;
    
    form {
      display:flex;
      flex-direction:column;
      align-items:center;
      justify-content:space-between;
      height:100%;
    }
}
```